src/bench/rpc_mempool.cpp

```diff
@@ -12,22 +12,23 @@
 #include <list>
 #include <vector>
 
+extern CTxMemPool mempool;
 
-static void AddTx(const CTransactionRef &tx, const CAmount &nFee, CTxMemPool &pool)
```

Comment:
why this change? Is this due to the fact that passing mempool reference to `AddTx` invalidate the validity of this bench?

Reply:
when passing around a mempool object, due to the way some other functions work, the test always returned an empty json object. using the global mempool object fixes that issue